src/java.base/share/classes/java/lang/invoke/StringConcatFactory.java

```diff
@@ -118,7 +117,6 @@ public final class StringConcatFactory {
      *
      * @since 21
      */
-    @PreviewFeature(feature=PreviewFeature.Feature.STRING_TEMPLATES)
     public static final int MAX_INDY_CONCAT_ARG_SLOTS;
```

Comment:
May this value change in the future? If yes, we might change this to a method to avoid incorrect eager inlining by the java compiler, or drop this with string templates.

Reply:
@lahodaj - I understand that - but I reverted to use the initializer, since the initializer was used before string templates (when the field was private).